src/elements/datepicker/datepicker-next-day/datepicker-next-day.visual.spec.ts

```diff
@@ -1,64 +1,62 @@
+import { sendKeys } from '@web/test-runner-commands';
 import { html, nothing } from 'lit';
 
+import { tabKey } from '../../core/testing/private/keys.js';
 import {
   describeEach,
   describeViewports,
   visualDiffDefault,
-  visualDiffFocus,
+  visualRegressionFixture,
 } from '../../core/testing/private.js';
-import './datepicker-next-day.js';
 
+import './datepicker-next-day.js';
 import '../datepicker.js';
 import '../../form-field.js';
 
 describe(`sbb-datepicker-next-day`, () => {
-  const cases = [
-    { name: 'no value', value: null },
-    { name: 'with value', value: '15.02.2023' },
-  ];
+  const cases = {
+    negative: [true, false],
```

Comment:
i suppose negative=true would require a black background color and a white outline color